content/en/docs/combined-matrix.md

```diff
@@ -0,0 +1,342 @@
+
+| Provider      | Deployment target                                               | Deployment strategies | Armory      | Notes |
+|---------------|-----------------------------------------------------------------|-----------------------|------------------------|-------|
+| Amazon AWS    | EC2, ECS, EKS                                                   |                       | All supported versions |       |
+| Cloud Foundry | PKS <ul><li>Versions A.B - X.Y</li></ul>                        |                       | All supported versions |       |
+| Google Cloud  | App Engine, Compute Engine, GKE                                 |                       | All supported versions |       |
+| Kubernetes    | Manifest-based deployments <ul><li>Versions A.B - X.Y</li></ul> |                       | All supported versions |       |
+## Dynamic accounts
+
+[![Generally available](/images/ga.svg)]({{< ref "release-definitions#ga" >}}) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+Dynamic accounts (external account configurations) for Spinnaker allow you to manage account configuration outside of Spinnaker, including secrets. Armory supports the following backend providers:
+
+#### Backend provider
+
+| Provider | Version | Armory | Notes |
+|---------|---------|---------|---------|
+| Git  | x.y.z | All supported versions |
+| S3  | n/a | All supported versions |
+| Vault  | x.y.z | All supported versions |
+
+#### Supported Spinnaker services
+
+| Service     | Account types               | Note                                                                                                           |
+|-------------|-----------------------------|----------------------------------------------------------------------------------------------------------------|
+| Clouddriver | Cloud provider, artifact    | Automatic configuration refreshing is supported for Cloud Foundry and Kubernetes cloud provider accounts only. |
+| Echo        | Pub/Sub                     |                                                                                                                |
+| Igor        | CI systems, version control |                                                                                                                |
+
+## External storage
+
+[![Generally available](/images/ga.svg)]({{< ref "release-definitions#ga" >}}) ![OSS](/images/oss.svg) ![Armory](/images/armory.svg)
+
+Spinnaker requires an external storage provider for persisting application settings and pipelines. Armory supports the following storage solutions:
+
+| Provider              | Armory      | Notes |
+|-----------------------|------------------------|-------|
+| Azure Storage         | All supported versions |       |
+| Google Cloud Storage  | All supported versions |       |
+| Minio                 | All supported versions |       |
```

Comment:
Note that we support usage of minio vs. minio itself - might be a critical thing if we start getting "My minio is down!"